src/resolvers/generate.ts

```
@@ -0,0 +1,740 @@
+                          .knex('${descendantModel.name}')
+                          .where({ id: descendant.id })
+                          .update({
+                            [\`${field.name}Id\`]: null,
+                          });
+                      });
+                    }
+                  }
+                `);
+                break;
+              }
+              case 'cascade':
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)